hadar/lib/services/authentication/LogInPage.dart

```diff
@@ -151,6 +155,18 @@ class _LogInPageState extends State<LogInPage> {
 
 
                       Widget curr_widget = await CurrentUser.init_user();
+
+                      //in case of deleted user
+                      var is_deleted_by_admin = curr_widget;
+                      if(is_deleted_by_admin == null){
+                        await FirebaseAuth.instance.signOut();
+                        setState(() {
+                          _error = ' החשבון שלך עדיין הוסר על ידי המנהל';
```

Comment:
I don't understand the `עדיין` in this sentence.

Reply:
my bad i will fix it and the rest of issues and i will merge it to main branch